source/extensions/filters/http/ratelimit/ratelimit.h

```diff
@@ -101,16 +103,19 @@ class FilterConfigPerRoute : public Router::RouteSpecificFilterConfig {
 public:
   FilterConfigPerRoute(
       const envoy::extensions::filters::http::ratelimit::v3::RateLimitPerRoute& config)
-      : vh_rate_limits_(config.vh_rate_limits()) {}
+      : vh_rate_limits_(config.vh_rate_limits()), rate_limit_policy_(config.rate_limits()) {}
 
   envoy::extensions::filters::http::ratelimit::v3::RateLimitPerRoute::VhRateLimitsOptions
   virtualHostRateLimits() const {
     return vh_rate_limits_;
   }
 
+  const Router::RateLimitPolicy& rateLimitPolicy() const { return rate_limit_policy_; }
+
 private:
   const envoy::extensions::filters::http::ratelimit::v3::RateLimitPerRoute::VhRateLimitsOptions
       vh_rate_limits_;
+  const FilterRateLimitPolicyImpl rate_limit_policy_;
```

Comment:
Do you care that it's specifically FilterRateLimitPolicy? The whole point is that you shouldn't care, just RateLimitPolicy should work. Will just need to specify the type in actual constructor, thus you'd need to write a real constructor in .cc file, and then actually write rate_limit_policy_ = FilterRateLimitPolicy(config.rate_limits())